test/functional/validation.regression.test.ts

```diff
@@ -81,7 +80,7 @@ suite("Validation regression tests", () => {
                 "Warning: The variable 'NSGRules' is never used. (arm-template (expressions))"])
     );
 
-    testWithRealFunctionMetadata(
+    testWithLanguageServerAndRealFunctionMetadata(
```

Comment:
This test uses the new test function - requires lang server and real function metadata to run.